golang/target/go.go

```diff
@@ -51,7 +52,7 @@ func (Go) E2eBuild(ctx context.Context) error {
 
 // Run builds binary and executes it
 func (Go) Run(ctx context.Context) error {
-	info, err := golang.WithSHA(golang.Build(ctx, BuildTarget))
+	info, err := golang.WithSHA(golang.Build(ctx, BuildTarget, ExtraBuildArgs...))
```

Comment:
the idea is to allow to add for instance `-tags=localrun` or similar things when executing code locally.